compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/gestures/MouseWheelScrollable.kt

```diff
@@ -65,7 +64,13 @@ internal class MouseWheelScrollNode(
         }
     }
 
-    private val pointerInputNode = delegate(SuspendingPointerInputModifierNode {
+    // Note that when `MouseWheelScrollNode` is used as a delegate of `ScrollableNode`,
+    // pointerInputNode does not get dispatched pointer events to in the standard manner because
+    // Modifier.Node.dispatchForKind does not dispatch to child/delegate nodes of the matching type,
+    // and `ScrollableNode` is already an instance of `PointerInputModifierNode`.
+    // This is worked around by having `MouseWheelScrollNode` simply forward the corresponding calls
+    // to pointerInputNode (hence its need to be `internal`).
```

Comment:
I'm not 100% sure about adding this comment.

On the one hand, I would find this useful because the behavior of `Modifier.Node.dispatchForKind` was surprising to me. On the other hand, this forwarding of events appears to be a common pattern. It's also used in Clickable, Draggable, StylusHandwritingNode, TextFieldDecoratorModifierNode etc.